### PR TITLE
Add compat data for contentScripts API

### PR DESCRIPTION
**Pull request by wbamberg**
This API is new in Firefox 59, and is for desktop and Android:

https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/contentScripts

### CODE REVIEW

**Comment by wbamberg** [comment]
In a previous PR for a new API ([pkcs11](https://github.com/mdn/browser-compat-data/pull/575)) it was requested that I should add data for `status`. I haven't done this here, because with https://github.com/mdn/browser-compat-data/issues/618 not resolved I don't know what the answer should be.

File: webextensions/api/contentscripts.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+              },
+              "firefox_android": {
+                "version_added": "59"
+              },
+              "opera": {
+                "version_added": false
+              }
+            }
+          },
+          "unregister": {
```
Comment (teoli2003) [inline webextensions/api/contentscripts.json]: Mmmh, not 100% sure, but I think register and unregister are 1 level too deep. @wbamberg, can you double check this?

Comment (wbamberg) [reply webextensions/api/contentscripts.json]: I don't think so: `register()` is a top level function in the namespace, but `unregister()` is a method of the `RegisteredContent Script object. So they are at different levels:

```
    "webextensions":
        "api"
            "contentScripts"
                "RegisteredContentScript"
                    "__compat"
                    "unregister"
                "register"
```

Running `$ npm run render webextensions.api.contentScripts 10 true > ~/Desktop/test.html 
` gives me the expected table:

screen shot 2018-01-15 at 9 08 32 am

Does that make sense, or am I missing something here?

But, this made me look again at this PR and I realised I got the case of the filename wrong, so I have corrected that.

Comment (teoli2003) [reply webextensions/api/contentscripts.json]: It does make sense. Thanks!

**Pull request merged**